tests/e2e/payout.bats

```diff
@@ -61,12 +61,64 @@ teardown_file() {
   [[ $(cached_pending_income) == 200000000 ]] || exit 1;
 }
 
+@test "payout: medium-priority queue processes payout and it is cancelled successfully" {
```

Comment:
What do these e2e tests add in terms of testing?

Simply extending an enum shouldn't require an e2e test to validate. A unit test should be plenty.